index.js

```diff
@@ -6,18 +6,18 @@ module.exports = {
 		dashicon: 'palmtree',
 		version: '1.0.0',
 		npmDependencies: [
-			"@wordpress/icons@^9.22.0",
+			"@wordpress/icons@^10.13.0",
 			"lodash@^4.17.21",
 		],
 		customScripts: {
 			"env": "wp-env"
 		},
 		npmDevDependencies: [
-			"@woocommerce/dependency-extraction-webpack-plugin@^2.2.0",
-			"@woocommerce/eslint-plugin@^2.2.0",
-			"@wordpress/prettier-config@^2.14.0",
-			"@wordpress/scripts@^26.2.0",
-			"@wordpress/env@^7.0.0"
+			"@woocommerce/dependency-extraction-webpack-plugin@^3.0.1",
+			"@woocommerce/eslint-plugin@^2.3.0",
+			"@wordpress/prettier-config@^4.13.0",
+			"@wordpress/scripts@^30.6.0",
```

Comment:
```suggestion
			"@wordpress/scripts@^30.7.0",
```

Nitpic: A few hours, version [`30.7.0`](https://www.npmjs.com/package/@wordpress/scripts) has been released.